lighthouse-core/audits/uses-rel-preconnect.js

```diff
@@ -135,6 +136,12 @@ class UsesRelPreconnectAudit extends Audit {
     const preconnectLinks = artifacts.LinkElements.filter(el => el.rel === 'preconnect');
     const preconnectOrigins = new Set(preconnectLinks.map(link => URL.getOrigin(link.href || '')));
 
+    if (preconnectLinks.length >= 6) {
+      return {
+        score: 1, warnings: preconnectLinks.length >= 8
```

Comment:
anyone on core team know of docs that could guide these numbers better than gut feel on a random comment? 😆 

```suggestion
        score: 1, warnings: preconnectLinks.length > 8
```

if we're sticking to our "more than 8" comment above :)

Reply:
> anyone on core team know of docs that could guide these numbers better than gut feel on a random comment? 😆

some docs for this (or even a cs.chromium.org reference) would be very appreciated